src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+
+        Machine[] machines = new Machine[] {truck, bulldozer, excavator};
+        for (Machine machine : machines) {
+            System.out.println(machine.doWork());
+            System.out.println(machine.stopWork());
```

Comment:
<img width="744" alt="Screen Shot 2022-11-08 at 5 54 00 PM" src="https://user-images.githubusercontent.com/90692308/200613097-ee69ffc8-d876-43a1-89a2-bd1eb8e193d4.png">
You should use System.out.println in the methods themselves